boards/nxp/frdm_rw612/frdm_rw612_common.dtsi

```diff
@@ -33,6 +35,16 @@
 			gpios = <&hsgpio0 12 0>;
 		};
 	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button_0: button_0 {
```

Comment:
newline

Reply:
Can you please give more context?
Add a new line, remove an extra new line?
I am not seeing where the issue is?

Reply:
add a newline between:
```
		compatible = "gpio-keys";

		user_button_0: button_0 {
```